tutorials/bert_e2e.py

```diff
@@ -0,0 +1,210 @@
+        [i.capitalize() for i in onnx_config.task.split('-')])
+    add_ppp.transformers_and_bert(bert_onnx_model, output_model_file,
+                                  vocab_file, tokenizer_type, 
+                                  task_type,
+                                  add_debug_before_postprocessing=True)
+
+
+def verify_results_for_e2e_model(model_name: str, input_bert_model: Path, output_model_file: Path):
+    """
+    Args:
+        output_model_file: the onnx model which finalized and needs to be verified
+        model_name: the huggingface model name
+        input_bert_model: the onnx model which is generated by huggingface or user provide
+    """
+    tokenizer, hg_model, _, text = get_tokenizer_and_model_from_huggingface(
+        model_name)
+    encoded_input = tokenizer(*text, return_tensors="pt")
+    transformers.set_seed(42)
+
+    session_options = onnxruntime.SessionOptions()
+
+    output_name_for_verify = ''
+    session = onnxruntime.InferenceSession(
+        str(input_bert_model.resolve(strict=True)), providers=["CPUExecutionProvider"]
+    )
+    inputs = {key: value.detach().numpy()
+              for key, value in encoded_input.items()}
+    output_name_for_verify = session.get_outputs()[0].name
+    ref_outputs = session.run([output_name_for_verify], inputs)
+
+    # Load tokenizer op
+    session_options.register_custom_ops_library(
+        onnxruntime_extensions.get_library_path())
+
+    session = onnxruntime.InferenceSession(
+        str(output_model_file.resolve(strict=True)), session_options, providers=["CPUExecutionProvider"]
+    )
+
+    inputs = dict(
+        input_text=np.array([[*text]]),
+    )
+    real_outputs = session.run([output_name_for_verify+"_debug"], inputs)
+    assert np.allclose(
+        real_outputs[0], ref_outputs[0], atol=1e-2, rtol=1e-6
+    ), f"Results do not match, expected:{ref_outputs[0]}, but got {real_outputs[0] }"
+
+    print("Results matches:",
+          real_outputs[0], "\ndiff:", real_outputs[0] - ref_outputs[0])
+
+
+def main():
+    parser = argparse.ArgumentParser(
+        os.path.basename(__file__),
+        description="""Add pre and post processing to a model.
+
+        This tutorial supports updating:
+          - MobileBert with different tasks
+          - XLM-Roberta with classification task   
+          
+        This tutorial does only server as a example of how to add pre/post processing to a transformer model.
+        It adds tokenizer( sentencePiece/Berttokenizer/HfbertTokenizer in pre-processing) 
+        and classify/Decoder (in postprocessing, serves for different downstream tasks) to the model.
+        
+        Export models from huggingface by default if there is not a onnx model provided.
+        NOTE: if provided a onnx model, you have to make sure your model is matched with the model_type in hugging-face.
+        As we are using the hugging-face tokenizer to do the pre-processing.
```

Comment:
```suggestion
        This tutorial provides an example of how to add pre/post processing to a transformer model.
        It can add a tokenizer (SentencePiece/Berttokenizer/HfbertTokenizer) for pre-processing,
        and a classifier/decoder for post-processing.
        
        Exports models from huggingface by default if an existing onnx model is not provided.
        NOTE: if providing a onnx model, you have to make sure your model is matched with the model_type in hugging-face as we are using the hugging-face tokenizer to do the pre-processing.
```